azext_iot/_params.py

```diff
@@ -119,3 +120,43 @@ def load_arguments(self, _):
         c.argument('labels', options_list=['--labels', '-lab'],
                    help="""Map of labels to be applied to target configuration.
                            Use the following format:'{\"key0\":\"value0\", \"key1\":\"value1\"}'""")
+
+    with self.argument_context('iot dps') as c:
+        c.argument('dps_name', help='Name of the Azure IoT Hub device provisioning service')
+        c.argument('initial_twin_properties', options_list=['--initial-twin-properties', '-props'],
+                   help='Initial twin properties')
+        c.argument('initial_twin_tags', options_list=['--initial-twin-tags', '-tags'],
+                   help='Initial twin tags')
+        c.argument('iot_hub_host_name', help='Host name of target IoT Hub')
+        c.argument('provisioning_status', arg_type=get_enum_type(EntityStatusType),
+                   help='Enable or disable enrollment entry')
+
+    with self.argument_context('iot dps enrollment') as c:
+        c.argument('enrollment_id', help='ID of device enrollment record')
+        c.argument('device_id', help='IoT Hub Device ID')
+        
+    with self.argument_context('iot dps enrollment create') as c:
+        c.argument('attestation_type', arg_type=get_enum_type(AttestationType), help='Attestation Mechanism')
+        c.argument('certificate_path', options_list=['--certificate-path', '-p'], 
+                   help='The path to the file containing the certificate. When choosing x509 as attestation type, certificate path is required.')
+        c.argument('endorsement_key', options_list=['--endorsement-key', '-k'], 
```

Comment:
For consistency in style, for multi word parameter names like '--endorsement-key' , consider changing the short name to the first letter of each word like '-ek'.